python/mxnet/ndarray/numpy/_op.py

```diff
@@ -178,6 +180,68 @@ def minimum(x1, x2, out=None):
     return _ufunc_helper(x1, x2, _npi.minimum, _np.minimum, _npi.minimum_scalar, None, out)
 
 
+@set_module('mxnet.ndarray.numpy')
+def mean(a, axis=None, dtype=None, out=None, keepdims=False):  # pylint: disable=arguments-differ
+    """
+    mean(a, axis=None, dtype=None, out=None, keepdims=None)
```

Comment:
Here `out` should be the last parameter. Please double check it. And here are additional two parameters, `name` and `initial`.

Reply:
Please double check whether the argument should be named from the fourth.

Reply:
I have wrapped this function to only expose the above signature